consensus/bor/bor.go

```diff
@@ -123,7 +123,7 @@ var (
 type SignerFn func(accounts.Account, string, []byte) ([]byte, error)
 
 // ecrecover extracts the Ethereum account address from a signed header.
-func ecrecover(header *types.Header, sigcache *lru.ARCCache) (common.Address, error) {
+func ecrecover(header *types.Header, sigcache *lru.ARCCache, c *params.BorConfig) (common.Address, error) {
```

Comment:
Could we do instead:
```
func (b *Bor) ecrecover()...
```

Reply:
no
because snapshot.go doesn't have bor object